sources/packages/web/src/components/aest/students/assessment/AwardTable.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-table class="bordered">
+    <thead>
+      <tr>
+        <th scope="col" class="text-left">Loan/grant type</th>
+        <th scope="col" class="text-left">Estimated award</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Canada Student Loan</td>
+        <td>
+          {{ awardDetails[`${identifier}cslf`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>B.C. Student Loan</td>
+        <td>
+          {{ awardDetails[`${identifier}bcsl`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          CSG-P
+          <tooltip-icon
+            >Canada Student Grant with Permanent Disability.</tooltip-icon
+          >
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}csgp`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          CSG-D
+          <tooltip-icon>Canada Student Grant with dependant(s).</tooltip-icon>
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}csgd`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          CSG-F
+          <tooltip-icon
+            >Canada Student Grant for Full time studies.</tooltip-icon
+          >
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}csgf`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          CSG-TU
+          <tooltip-icon>Canada Student Grant for Top-up.</tooltip-icon>
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}csgt`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          BCAG
+          <tooltip-icon>British Colombia Access Grant.</tooltip-icon>
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}bcag`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          BCAG-D
+          <tooltip-icon
+            >British Colombia Access Grant with disabilities.</tooltip-icon
+          >
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}bcagd`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          SBSD
+          <tooltip-icon
+            >British Colombia Supplemental Bursary with
+            Disabilities.</tooltip-icon
+          >
+        </td>
+        <td>
+          {{ awardDetails[`${identifier}sbsd`] ?? NOT_ELIGIBLE }}
+        </td>
+      </tr>
+    </tbody>
+  </v-table>
+</template>
+<script lang="ts">
+import { DynamicAwardDTO } from "@/services/http/dto";
+import { PropType } from "vue";
+
+export default {
+  props: {
+    awardDetails: {
+      type: Object as PropType<DynamicAwardDTO>,
+      required: true,
+      default: {} as DynamicAwardDTO,
+    },
+    identifier: {
```

Comment:
for my info, what is the purpose of this prop

Reply:
This the prefix that combines with the award types to form the dynamic variable. 
e.g when identifier = disbursement1 -> disbursement1cslf, disbursement1bcag
The identifiers we assign are disbursement1, disbursement2, disbursementReceipt1, disbursementReceipt2

Reply:
I also thought the same but that will introduce changes to existing noa. 
I am just thinking is it worth the effort as this dynamic values is only for noa and this page.

Reply:
Just to add some points, when we create the dynamic DTO for the NOA was with the idea that the form.io could be changed to accommodate a new grant, for instance, and it would not require a code change.
Since we are moving the UI to Vuetify (I hope that NOA page will be Vuetify) maybe we can review if we still need these DTOs to be dynamic.
Apparently, even the schedule can be considered always as 1 or 2, so maybe the DTO can be more specific. Either way it does not seems to be conversation that would change this PR 😉